src/main/java/org/wiremock/integrations/testcontainers/WireMockContainer.java

```diff
@@ -15,28 +15,23 @@
  */
 package org.wiremock.integrations.testcontainers;
 
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.containers.wait.strategy.WaitStrategy;
+import org.testcontainers.images.builder.Transferable;
+import org.testcontainers.shaded.com.google.common.io.Resources;
+import org.testcontainers.utility.MountableFile;
+
 import java.io.File;
 import java.io.IOException;
 import java.net.URL;
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
It goes against the code style declared in the project. I do not feel strongly against it but better to not do so https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
Okay I see, I will adjust that IntelliJ tried to optimize...